lib/digital_public_works_web/plugs/meta.ex

```diff
@@ -0,0 +1,22 @@
+defmodule DigitalPublicWorks.Plugs.Meta do
+  import Plug.Conn
+
+  def init(opts), do: opts
+
+  def call(conn, _params) do
+    assign(conn, :meta, %{
+      title: "Digital Public Works",
+      description: "A web platform for connecting civic projects with volunteers and mentors",
+      keywords: "civic tech software programming",
+      "og:url": build_url(conn)
+    })
+  end
+
+  defp build_url(%Plug.Conn{scheme: scheme, host: host, port: port, request_path: path}) do 
```

Comment:
This has the side effect of including port 443 in all urls (at least in the Heroku review app). We may want to consider just dropping the port since I don't think we will need to point to a non-standard port.

Reply:
Yeah, i wasnt sure if the port was going to be populated. This one was a bit of a tracer round. 

Will remove since having the port really only was to support localhost dev being correct which is not super relevant.